tests/e2e/instrumentation-nodejs/00-install-collector.yaml

```diff
@@ -4,8 +4,6 @@ metadata:
   name: sidecar
 spec:
   mode: sidecar
-  args:
```

Comment:
Instead of removing args here, I should suggest adding them into the `service` field where the latest changes expect the args. Having extra config is always plus rather than dropping them from existing tests.

Reply:
@pavolloffay and @jpkrohling mentioned the same in a previous comment, however, we agreed to create an issue(#702) to create a new test checking the new approach of those flags.